### PR TITLE
Fix updating teamranks

### LINKED ISSUE #6107: Teamranks not inserted at all
Screenshot 2022-12-08 at 10 09 22

For example the new top rank on TsinmaS.

I also can't find them in the _backup table.

Logs:
```
2022-12-08 06:33:58 I chat: *** Cireme finished in: 2 minute(s)  6.60 second(s)
2022-12-08 06:33:58 I chat: *** New record: 10.48 second(s) better.
2022-12-08 06:33:58 I chat: *** Meow~ finished in: 2 minute(s)  6.60 second(s)
2022-12-08 06:33:58 I chat: *** New record: 10.48 second(s) better.
2022-12-08 06:33:58 I sql: SQLite statement: INSERT OR IGNORE INTO record_race_backup(	Map, Name, Timestamp, Time, Server, 	cp1, cp2, cp3, cp4, cp5, cp6, cp7, cp8, cp9, cp10, cp11, cp12, cp13, 	cp14, cp15, cp16, cp17, cp18, cp19, cp20, cp21, cp22, cp23, cp24, cp25, 	GameID, DDNet7) VALUES ('TsinmaS', 'Cireme', DATETIME('2022-12-08 06:33:58', 'utc'), 126.60, 'GER', 	0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 	0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 	0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 	'4f465b82-9052-4d84-a5c9-92efe793328f', 0)
2022-12-08 06:33:58 I sql: [237] save score done on write backup database, Success=1
2022-12-08 06:33:58 I sql: SQLite statement: INSERT OR IGNORE INTO record_race_backup(	Map, Name, Timestamp, Time, Server, 	cp1, cp2, cp3, cp4, cp5, cp6, cp7, cp8, cp9, cp10, cp11, cp12, cp13, 	cp14, cp15, cp16, cp17, cp18, cp19, cp20, cp21, cp22, cp23, cp24, cp25, 	GameID, DDNet7) VALUES ('TsinmaS', 'Meow~', DATETIME('2022-12-08 06:33:58', 'utc'), 126.60, 'GER', 	0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 	0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 	0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 	'4f465b82-9052-4d84-a5c9-92efe793328f', 0)
2022-12-08 06:33:58 I sql: [238] save score done on write backup database, Success=1
2022-12-08 06:33:58 I sql: SQLite statement: INSERT OR IGNORE INTO record_teamrace_backup(Map, Name, Timestamp, Time, ID, GameID, DDNet7) VALUES ('TsinmaS', 'Cireme', DATETIME('2022-12-08 06:33:58', 'utc'), 126.60, x'77ab2a60fd894120a62b407ec623754f', '4f465b82-9052-4d84-a5c9-92efe793328f', 0)
2022-12-08 06:33:58 I sql: SQLite statement: INSERT OR IGNORE INTO record_teamrace_backup(Map, Name, Timestamp, Time, ID, GameID, DDNet7) VALUES ('TsinmaS', 'Meow~', DATETIME('2022-12-08 06:33:58', 'utc'), 126.60, x'77ab2a60fd894120a62b407ec623754f', '4f465b82-9052-4d84-a5c9-92efe793328f', 0)
2022-12-08 06:33:58 I sql: [239] save team score done on write backup database, Success=1
2022-12-08 06:33:58 I sql: [237] save score done on write database
2022-12-08 06:33:58 I sql: [238] save score done on write database
2022-12-08 06:33:58 I sql: found team rank from same team (old time: 177.979996, new time: 126.599998)
2022-12-08 06:33:58 I sql: [239] save team score done on write database
```
Edit: Ok, found them in the backup sqlite table, let me check why my script isn't picking them up.
Edit2: Was because ssh connection to server failed, got them from backup table now.

### PR DESCRIPTION
**Pull request by Zwelf**
Fixes #6107

## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [x] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/game/server/scoreworker.cpp [hunk 1]
```diff
@@ -640,23 +641,23 @@ bool CScoreWorker::SaveTeamScore(IDbConnection *pSqlServer, const ISqlData *pGam
...
 				str_format(aBuf, sizeof(aBuf),
-					"UPDATE %s_teamrace SET Time=%.2f, Timestamp=?, DDNet7=%s, GameID=? WHERE ID = ?",
-					pSqlServer->GetPrefix(), pData->m_Time, pSqlServer->False());
+					"UPDATE %s_teamrace SET Time=%.2f, Timestamp=%s, DDNet7=%s, GameID=? WHERE ID = ?",
+					pSqlServer->GetPrefix(), pData->m_Time, pSqlServer->InsertTimestampAsUtc(), pSqlServer->False());
 				if(pSqlServer->PrepareStatement(aBuf, pError, ErrorSize))
 				{
 					return true;
 				}
 				pSqlServer->BindString(1, pData->m_aTimestamp);
```
Comment (def-) [inline src/game/server/scoreworker.cpp:650]: We are still binding the timestamp here?

Comment (Zwelf) [reply src/game/server/scoreworker.cpp:650]: Yes:
* https://github.com/ddnet/ddnet/blob/5290193f1fe3f9fb91345fb95d4b35a027acbd13/src/engine/server/databases/mysql.cpp#L75
* https://github.com/ddnet/ddnet/blob/5290193f1fe3f9fb91345fb95d4b35a027acbd13/src/engine/server/databases/sqlite.cpp#L21
The timestamp is read nowhere. Which probably made this go unnoticed. And only affects updated teamranks in sqlite.

Comment (Zwelf) [reply src/game/server/scoreworker.cpp:650]: Although it is indeed a bit unrelated. I can split it up into another commit or PR if wanted.

**Comment by def-** [comment]
bors r+
Thanks for the quick fixes Zwelf.

**Pull request merged**